packages/@divvi/mobile/src/public/components/Button.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import BaseButton, { BtnSizes, BtnTypes, ButtonProps, TextSizes } from 'src/components/Button'
+
+export { BtnSizes, BtnTypes, TextSizes }
```

Comment:
I'd like not to have `enums` on the public interface.
See https://www.typescriptcourse.com/string-literal-unions-over-enums

Could you do a small "wrapping" too?
Something like:

```tsx
export type ButtonSize = 'primary' | 'secondary' | 'tertiary'`
export type ButtonType = ...
export type ButtonTextSize = ...
```

Reply:
Should I convert these enums throughout to string literal unions throughout?

Reply:
I ended up doing something different, which results in some mildly unuseful docs, but it has the advantage of remaining up to date if new types are added to the base button. I am open to other suggestions.